### PR TITLE
feat: autocapture.elementInteractions (visual labeler)

### PR DESCRIPTION
**Pull request by Mercy811**
- deprecate `defaultTracking` with `autocapture`
- merge string list and regex list of `pageUrlAllowlist` and `dataAttributePrefix` to support `configuration.autocapture.elementInteractions`

### CODE REVIEW

File: src/amplitude-wrapper.js [hunk 1]
```diff
@@ -155,14 +155,42 @@ var amplitudeUserAgentEnrichmentPlugin=function(i){"use strict";var e=function()
...
+          ...(configuration.autocapture.attribution.excludeReferrersText || []),
+          ...(configuration.autocapture.attribution.excludeReferrersRegex?.map(item => new RegExp(item)) || [])
         ];
-        delete configuration.defaultTracking.attribution.excludeReferrersText;
-        delete configuration.defaultTracking.attribution.excludeReferrersRegex;
-        configuration.defaultTracking.attribution.excludeReferrers = excludeReferrers;
+        delete configuration.autocapture.attribution.excludeReferrersText;
+        delete configuration.autocapture.attribution.excludeReferrersRegex;
+        // if excludeReferrers is empty, use default
+        if (excludeReferrers.length !== 0) {
```
Comment (yuhao900914) [inline src/amplitude-wrapper.js:166]: Is it still possible that customer provide empty excludeReferersText and excludeReferrersRegex to overwrite the default excludeReferrers?

Comment (Mercy811) [reply src/amplitude-wrapper.js:166]: This is to fix the bug of the default behavior that when nothing is set, `excludeReferrers` is empty and then it override the default exclude referrers which is the top level domain. To fix it, we should not set `excludeReferrers` to an empty list. However, it's then not possible to set it as an empty list. But I think it's fine that, if customers want to include all referrer, they can just set a special value here.

Comment (yuhao900914) [reply src/amplitude-wrapper.js:166]: can we check real empty and [] and treat them differently?

Comment (Mercy811) [reply src/amplitude-wrapper.js:166]: There is no difference between real empty and []

**Review by yuhao900914 — APPROVED** [review]

**Pull request merged**